.gitignore

```diff
@@ -0,0 +1,46 @@
+# Logs
```

Comment:
I don't have a problem with adding the `.gitignore` but there are sections in here that shouldn't be, such as ProcBot secrets.

Reply:
Remove ProcBot.pem, anything else?